chamber/multidomains/auth/backends.py

```diff
@@ -1,10 +1,26 @@
-from django.contrib.auth.backends import ModelBackend as OridginModelBackend
+from django.contrib.auth.backends import ModelBackend as OriginModelBackend
 from django.contrib.auth.models import Permission
 
 from chamber.multidomains.domain import get_user_class
 
+from auth_token.backends import DeviceBackend as OriginDeviceBackend
```

Comment:
For me this is surprising it works, beacause I can't see auth token in project requirements.

Reply:
Upraveno - vyhovuje?